Automattic/jetpack
# Add player to media selector

#### Changes proposed in this Pull Request:

* Adds a video player to the Media selector in the block editor

#### Other information:


#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Add a VideoPress block
* Click "Media Library"
* Click on a VideoPress video that was uploaded before, and check if the video is rendered on the right side bar:
![2022-08-31_15-27](https://user-images.githubusercontent.com/971483/187753881-5775fbaf-ad86-471e-a703-babc2db2e1b2.png)


* Check that other non-videopress video still work without any changes
